util/mu.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  sparqlEscapeUri as mu_sparqlEscapeUri,
+  sparqlEscapeString as mu_sparqlEscapeString,
+  sparqlEscapeDateTime as mu_sparqlEscapeDateTime,
+} from 'mu';
+
+export function sparqlEscapeString(stringValue: string) {
+  if (!stringValue || typeof stringValue !== 'string') {
+    throw Error(
+      `Could not sparql escape string from passed value: ${stringValue}`,
+    );
+  }
```

Comment:
Extracted this to our own util method as all the errors I've seen had somehting to do with cannot call .replace on value. This could help with finding the error faster.

```bash
mandataris-1  | [500]: Could not sparql escape string from passed value: undefined
mandataris-1  | Error: Could not sparql escape string from passed value: undefined
mandataris-1  |     at sparqlEscapeString (/build/dist/util/mu.js:12:11)
mandataris-1  |     at checkAuthorization (/build/dist/routes/burgemeester-benoeming.js:73:52)
mandataris-1  |     at onBurgemeesterBenoeming (/build/dist/routes/burgemeester-benoeming.js:410:11)

```

Reply:
But could be overkill and actually the undefined should be fixed in the mu library itself